bl/policy/policy_factory.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/datreeio/datree/pkg/cliClient"
 	internal_policy "github.com/datreeio/datree/pkg/policy"
+	"github.com/xeipuuv/gojsonschema"
```

Comment:
We'll need matching test cases for the new validations under `/pkg/policy/policy_test.go`